src/PrestaShopBundle/Resources/views/Admin/Module/Includes/card.html.twig

```diff
@@ -9,7 +9,8 @@
      data-version="{{ module.attributes.version }}"
      data-description="{{ module.attributes.description }}"
      data-tech-name="{{module.attributes.name }}"
-     data-categories="{{ module.attributes.refs|join('|') }}"
+     data-child-categories="{{ module.attributes.categoryName }}"
+     data-categories="{{ module.attributes.categoryParent }}"
```

Comment:
javascript need to be updated to reflect the fact that module have one and only one category